ftplugin/latex-suite/folding.vim

```diff
@@ -257,6 +257,25 @@ function! MakeTexFolds(force, manual)
 	endif
 	" }}}
 
+	if g:Tex_FoldedMisc =~ '\<beamer\>'
+		" {{{ frame (in beamer)
+		call AddSyntaxFoldItem (
+					\ '^\s*\\frame',
+					\ '^\s*\\frame\|^\s*\\end{document}\|^\s*\\begin{frame}\|^\s*\\\(sub\)*section{',
+					\ 0,
+					\ -1,
+					\ )
+		" }}}
+		" {{{ column (in beamer)
+		call AddSyntaxFoldItem (
+					\ '^\s*\\column',
+					\ '^\s*\\column\|^\s*\\end{columns}',
```

Comment:
This fold should also end on `\begin{column}` if one mixes the macro `\column` and the environment `column`?

Reply:
Okay, I will add this.

Reply:
I suggest `'^\s*\\\%(column\|begin{column}\|end{columns}\)'` as an end pattern. Otherwise, `\end{column}` is not part of the fold. Note that `\%(` is preferred over `\(`, since it should be a little bit faster (if I recall the documentation right).

Reply:
I have done this, but note that with this regex, the parent `columns` environment will not fold. Maybe you prefer the parent environment not folding, but I have edited my folding file to fold the parent environment too. It's not a big deal though, we can adopt this practice, as this is the default for the rest of the plugin.

Reply:
I see. Yes, it should be a good idea, if also the columns environment will fold.

Reply:
Ok, done!